internal/clients/http/sources_errors.go

```diff
@@ -8,10 +8,11 @@ import (
 )
 
 var (
-	AuthenticationForSourcesNotFoundErr = fmt.Errorf("authentications for source weren't found in sources app: %w", clients.NotFoundErr)
-	ApplicationNotFoundErr              = fmt.Errorf("application not found in sources app: %w", clients.NotFoundErr)
-	ApplicationReadErr                  = fmt.Errorf("application read returned no application type in sources: %w", clients.NotFoundErr)
-	ApplicationTypeNotFoundErr          = fmt.Errorf("application type 'provisioning' not found: %w", clients.NotFoundErr)
-	SourceNotFoundErr                   = fmt.Errorf("source not found: %w", clients.NotFoundErr)
-	AuthenticationSourceAssociationErr  = errors.New("authentication associated to source id not found")
+	MoreThanOneAuthenticationForSourceErr = errors.New("more than one authentication")
```

Comment:
Have you rebased on top of "main"? Because this error is no longer in use. But it's fine we can merge and remove later.